client/src/routes/HomePage/HomePage.js

```diff
@@ -64,6 +65,15 @@ export default function HomePage(props) {
       .catch((err) => console.log(err));
   }, [show]);
 
+  useEffect(() => {
```

Comment:
This part doesn't work, because props is not being passed to the Home Page from the Outlet. It also doesn't work to set it based on when the local storage for collapsed changes, but I'm not sure why. It can access the variable and all, but doesn't rerender when the local storage is changed.

@piratecody @FletchDavid